app/src/main/res/values/strings.xml

```diff
@@ -60,123 +60,123 @@
     <string name="untracked">untracked</string>
 
     <string name="warning_title" description="Status indicator text shown in a warning notification.">
-    Warning: RethinkDNS has been deactivated
+    Warning: RethinkDNS turned off.
     </string>
 
     <string name="notification_content" description="Indicates what will happen if the user taps the notification.">
-    Tap to change protection setting
+    Tap to change protection setting.
   </string>
 
     <string name="dns_mode_notification_title">
-        Protected! Connected to Fast and Secure DNS.
+        Protected. Connected to fast and secure DNS.
     </string>
 
     <string name="firewall_mode_notification_title">
-        Protected! Connected to Firewall.
+        Protected. Connected to firewall.
     </string>
 
     <string name="hybrid_mode_notification_title">
-        Protected! Connected to DNS + Firewall.
+        Protected. Connected to DNS + firewall.
     </string>
 
-    <string name="pause_mode_notification_title">Paused! Blocked apps will continue to be firewalled.</string>
+    <string name="pause_mode_notification_title">Paused. Firewalling of blocked apps will continue.</string>
 
-    <string name="status_waiting" description="This text appears during the connection process if Intra needs user permission or the phone is offline.">
-    waiting
+    <string name="status_waiting" description="This text appears during the connection process if Intra needs user permission or the device is offline.">
+    Waiting…
     </string>
 
     <string name="status_failing" description="This text appears if the connection is currently not working correctly. [CHAR_LIMIT=NONE]">
-    failing
+    Failing.
     </string>
 
-    <string name="status_protected" description="This is a large message in a reassuring color.  It indicates that DNS queries can't be surveilled or modified in transit.">
-    protected
+    <string name="status_protected" description="This is a large message in a reassuring color. It indicates DNS queries can't be surveilled or modified in transit.">
+    Protected.
     </string>
 
-    <string name="status_app_error">app error</string>
+    <string name="status_app_error">App error.</string>
 
-    <string name="status_dns_error">dns error</string>
+    <string name="status_dns_error">DNS error.</string>
 
-    <string name="status_dns_server_down">dns server down</string>
+    <string name="status_dns_server_down">DNS server down.</string>
 
-    <string name="status_no_internet">no internet</string>
+    <string name="status_no_internet">Not connected to the Internet.</string>
```

Comment:
```suggestion
    <string name="status_no_internet">Not connected to the Internet</string>
```

Reply:
"No Internet" is what we want here. The "status" UI isn't that big to accommodate this longer string.